packages/shopify-codemod/transforms/global-reference-to-import.js

```diff
@@ -68,6 +69,13 @@ export default function globalReferenceToImport(
     return fileForIdentifier[identifier];
   }
 
+  function searchForNamedExports(filename) {
+    if (filename.endsWith('.coffee'))
+      return false;
+    const namedExports = j(fs.readFileSync(filename).toString())
```

Comment:
I think we can probably get by with a tacky version of this by using the same grepping stuff we use to find the identifier in the first place, but instead of searching for the identifier we just scope our search to only that file with something like `export\s(\S+)`. It's less accurate than doing the AST but should be fine since `export` is a reserved word. Check whether the export is default or not and you should get the same thing. Not sure if this will be faster but it's worth a try.